source/Halibut/Diagnostics/LogWriters/AggregateLogWriterLog.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections.Generic;
+
+namespace Halibut.Diagnostics.LogWriters
+{
+    /// <summary>
+    /// An ILog which writes the log events to both an inner ILog and to the set of ILogWriters.   
+    /// </summary>
+    public class AggregateLogWriterLog : ILog
```

Comment:
I'm finding the names of things a little confusing. LogWriterLog for example!!

Reply:
Dropped the `Log` from the end. Maybe that helps.

Reply:
For what it is worth it is unlikely that anyone needs to actually use this class directly.